src/upload/upload.handler.ts

```diff
@@ -21,6 +21,13 @@ async function unlinkFiles (files: any[]): Promise<void> {
 export default function (storageProvider: ProviderManager, libp2p: Libp2p): UploadRouteHandler {
   return async (req: any, res: any): Promise<void> => {
     const { offerId, peerId, account } = req.body
+    const missedParams = ['offerId', 'peerId', 'account'].filter(k => req.body[k])
+
+    if (!missedParams.length) {
+      return res.status(422).json({
```

Comment:
Are you sure it should be `422`? Shouldn't it be `400`?

Reply:
400 Bad Request
This response means that server could not understand the request due to invalid syntax.

422 Unprocessable Entity
The request was well-formed but was unable to be followed due to semantic errors.

400 code more for invalid request, but theoretically request is valid HTTP request

Reply:
I don't know

Reply:
During my days of web developer I think I have never seen `422` used...

For example here: https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/
The 400 is used there.

But then here: https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api
The 422 is used there.

🤔

Ok, lets keep it like this.